ingestion/src/metadata/sampler/processor.py

```diff
@@ -88,11 +88,7 @@ def _run(self, record: ProfilerSourceAndEntity) -> Either[SamplerResponse]:
                 schema_entity=schema_entity,
                 database_entity=database_entity,
                 table_config=get_config_for_table(entity, self.profiler_config),
-                default_sample_config=SampleConfig(
-                    profile_sample=self.source_config.profileSample,
-                    profile_sample_type=self.source_config.profileSampleType,
-                    sampling_method_type=self.source_config.samplingMethodType,
-                ),
+                default_sample_config=SampleConfig(),
```

Comment:
keeping simpler pipeline for the auto classification. If there's anything configured for the table we'll pick it from there directly